### ISSUE TITLE
I have another significant question!

### ISSUE THREAD

**Issue by qzxwj** [irvinxav/Efficient-HetConv-Heterogeneous-Kernel-Based-Convolutions#4]
Dear irvinxav:
Thanks for your code. Are you sure your picture in you paper illustrates the code correctly? I find that the parameter 'p' has no effect on the amount of the 1*1 Conv. Looking forward to your reasonable answer.

'''
class HetConv(nn.Module):
    def __init__(self, in_channels, out_channels, p):
        super(HetConv, self).__init__()
        # Groupwise Convolution
        self.gwc = nn.Conv2d(in_channels, out_channels, kernel_size=3, padding=1, groups=p, bias=False)
        # Pointwise Convolution
        self.pwc = nn.Conv2d(in_channels, out_channels, kernel_size=1, bias=False)

    def forward(self, x):
        return self.gwc(x) + self.pwc(x)
'''

![Image](https://github.com/user-attachments/assets/d72c5084-70d8-486f-b70e-48b2e24eda08)
